.github/workflows/frontend-check.yaml

```diff
@@ -14,6 +14,16 @@ jobs:
         run: podman build -t e2e-frontend -f frontend/e2e_frontend.containerfile ./frontend
       - name: run unit tests
         run: podman run --rm -it e2e-frontend run test:unit
+
+  component-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout project
+        uses: actions/checkout@v4   
```

Comment:
When did `checkout` go to v4? Hmm -- we've been using v3, and I never realized it was out of date. 😦